README.rst

```diff
@@ -332,8 +332,7 @@ base class).
             )
             # append ctrl to list
             self.ctrl_list.append(ctrl)
-            # flash and start terminal
-            ctrl.flash()
```

Comment:
I think it's still better to keep this since it will work for other BOARDs, and for native it still compiles the application anyway no?

Reply:
> I think it's still better to keep this since it will work for other BOARDs, and for native it still compiles the application anyway no?

Maybe a `ctrl.flash()` should restart the application at least with native after the rebuild?